bindings/python/test/flight/test_maneuver.py

```diff
@@ -0,0 +1,237 @@
+from ostk.physics.time import Scale
+from ostk.physics.coordinate import Frame
+from ostk.physics.unit import Mass
+
+from ostk.astrodynamics.flight import Maneuver
+from ostk.astrodynamics.dynamics import Tabulated as TabulatedDynamics
+
+
+@pytest.fixture
+def instants() -> list[Instant]:
+    return [
+        Instant.J2000(),
+        Instant.J2000() + Duration.seconds(1.0),
+        Instant.J2000() + Duration.seconds(5.0),
+        Instant.J2000() + Duration.seconds(7.0),
+    ]
+
+
+@pytest.fixture
+def acceleration_profile() -> list[np.ndarray]:
+    return [
+        np.array([1.0e-3, 0.0e-3, 0.0e-3]),
+        np.array([0.0e-3, 1.0e-3, 0.0e-3]),
+        np.array([0.0e-3, 0.0e-3, 1.0e-3]),
+        np.array([1.0e-3, 1.0e-3, 1.0e-3]),
+    ]
+
+
+@pytest.fixture
+def frame() -> Frame:
+    return Frame.GCRF()
+
+
+@pytest.fixture
+def mass_flow_rate_profile() -> list[float]:
+    return [1.0e-5, 1.1e-5, 0.9e-5, 1.0e-5]
+
+
+@pytest.fixture
+def maneuver(
+    instants: list[Instant],
+    acceleration_profile: list[np.ndarray],
+    frame: Frame,
+    mass_flow_rate_profile: list[float],
+) -> Maneuver:
+    return Maneuver(instants, acceleration_profile, frame, mass_flow_rate_profile)
+
+
+class TestManeuver:
+    def test_constructor(
+        self,
+        maneuver: Maneuver,
+    ):
+        assert maneuver is not None
+        assert isinstance(maneuver, Maneuver)
+        assert maneuver.is_defined()
+
+    def test_getters(
+        self,
+        maneuver: Maneuver,
+        instants: list[Instant],
+        acceleration_profile: list[np.ndarray],
+        mass_flow_rate_profile: list[float],
+    ):
+
+        assert maneuver.is_defined()
+
+        assert maneuver.get_instants() == instants
+
+        computed_acceleration_profile_default_frame = maneuver.get_acceleration_profile()
+        for i in range(len(instants)):
+            assert np.array_equal(
+                computed_acceleration_profile_default_frame[i], acceleration_profile[i]
+            )
+
+        computed_acceleration_profile_non_default_frame = (
+            maneuver.get_acceleration_profile(frame=Frame.ITRF())
+        )
+        for i in range(len(instants)):
+            assert (
+                np.array_equal(
+                    computed_acceleration_profile_non_default_frame[i],
+                    acceleration_profile[i],
+                )
+                == False
+            )
+
+        assert maneuver.get_mass_flow_rate_profile() == mass_flow_rate_profile
+
+    def test_instant_getters(
+        self,
+        maneuver: Maneuver,
+        instants: list[Instant],
+    ):
+        assert maneuver.get_start_instant() == instants[0]
+
+        assert maneuver.get_end_instant() == instants[-1]
+
+        assert maneuver.get_duration() == instants[-1] - instants[0]
+
+    def test_calculators(
+        self,
+        maneuver: Maneuver,
+    ):
+        pass
+        # assert maneuver.calculate_delta_v().get_value() == pytest.approx(
```

Comment:
TBD?

Reply:
Mentioned this in my comment to reviewers, waiting on releasing `Scalar` in python in OSTk phys